app/containers/uri/URILandingPage.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { ROUTES } from '../../routes-config';
+import { getUrlParameterByName } from '../../utils/routing';
+import type { InjectedProps } from '../../types/injectedPropsType';
+import URILandingDialogContainer from './URILandingDialogContainer'
+
+type Props = InjectedProps;
+
+// just for testing purposes. this logic doesn't belong here
```

Comment:
it’s good to write comments! if it’s a task could you add `// TODO: text here` and explain a little more how much logic is not supposed to be there and what is the proper way to do it. It would help with comments about how u are planning to do stuff 😄

Reply:
Understood! Thanks for the feedback ;) As you see there's little logic for now, but would like to know if in terms of structure it looks ok (to avoid too much refactoring later on).